src/components/FeatureViews/Accounts/ResetPasswordDialogs.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+
+import { ResponsiveButton } from "../../../theme/ThemedComponents/Button/ResponsiveButton";
+import { fullName } from "utils";
+
+
+const ResetPasswordDialogs = ({userInfo, open, openReset, password, handleClosePassword, handleClosePasswordReset, handleCloseReset}) => {
+
+    return (
+    <>
+      <Dialog
+        open={open}
+        onClose={handleClosePassword}
+        maxWidth="xs"
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle disableTypography id="alert-dialog-title" className="center dialog-padding">
+          Do you want to reset this user's password?
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description" className="center">
+            When you reset the password, a randomly generated password will
+            become the user's new password. This action cannot be reverted. An
+            automatic email will be sent out notifying the user that their
+            password has been reset. The user can also reset their own password
+            through the portal login page.
+          </DialogContentText>
+        </DialogContent>
+        <DialogActions className="dialog-actions">
+          <ResponsiveButton
+            onClick={handleClosePassword}
+            variant="outlined"
+            className="cancelBtn"
+          >
+            cancel
+          </ResponsiveButton>
+          <ResponsiveButton
+            onClick={handleClosePasswordReset}
+            className="resetBtn"
+            autoFocus
+          >
+            reset password
+          </ResponsiveButton>
+        </DialogActions>
+      </Dialog>
+      <Dialog
+        open={openReset}
+        onClose={handleCloseReset}
+        aria-labelledby="alert-dialog-title"
```

Comment:
ditto for this - rename this to something that helps the reader know what this dialog is used for

Reply:
Done! Thank you for this, it is so important to be accessible!